python/paddle/v2/fluid/layers/nn.py

```diff
@@ -847,7 +847,35 @@ def cos_sim(X, Y, **kwargs):
     return out
 
 
-def dropout(x, dropout_prob, is_test=False, seed=0, **kwargs):
+def dropout(x, dropout_prob, is_test=False, seed=None, **kwargs):
+    """
```

Comment:
it's better to set with 0. Our default value equals 0 in dropout op.

Reply:
If the seed is set to 0, it will be a fixed seed 0, not a random seed. So if I miss something?

Reply:
Here when the seed is set to None, a random seed will be used.